### DISCUSSION TITLE
umi-lib 打包Error: Unexpected character '�' (Note that you need plugins to import files that are not JavaScript)

### DISCUSSION THREAD

**Discussion by cater111** [umijs/umi#11017] [General]
![image](https://user-images.githubusercontent.com/26265046/233637715-b075271c-582d-4328-9e95-b3152f6f30a4.png)
![image](https://user-images.githubusercontent.com/26265046/233637787-4e57e588-d499-4119-acc2-8856cfe4f280.png)

**Comment by fz6m** [comment]
umi-lib 是什么。
